cluster/syncer/statefulset.go

```diff
@@ -105,14 +106,19 @@ func (s *StatefulSetSyncer) Sync(ctx context.Context) (syncer.SyncResult, error)
 
 	result.Operation, err = s.createOrUpdate(ctx)
 
+	//get namespace.
```

Comment:
key is `ObjectKey{Namespace: obj.GetNamespace(), Name: obj.GetName()}` , it contain namespace and name

Reply:
fixed.